tendrl/gluster_integration/objects/definition/gluster.yaml

```diff
@@ -961,6 +961,24 @@ namespace.gluster:
         quorum_status:
           help: "Quorum status"
           type: String
+        usable_capacity:
```

Comment:
`total_capacity` would be better here

Reply:
I would not change it in this PR, this was added as it was missing in defn file. This attr is being used in many places. Better to take later.